browser/src/view/components/NavigationMenu/NavigationMenu.js

```diff
@@ -9,12 +9,15 @@ export const NavigationMenu = ({ links }) => {
       <ul className='nav-user-actions'>
         {links.map(link => {
           return (
-            <Button
-              className=''
-              label={link.label}
-              style='icon-label'
-              icon={<link.Icon />}
-            />
+            <li key={link.label}>
+              <Button
+                className=''
+                layout='icon'
+                label={link.label}
+                Icon={<link.Icon />}
+                type='primary'
```

Comment:
If you go by the static property method, you'll be able to reference these by constant, allaying the worry of a wrongly input string.